src/Util/OlStyleUtil.ts

```diff
@@ -273,6 +297,208 @@ class OlStyleUtil {
 
     return template;
   }
+
+  public static evaluateFunction(func: GeoStylerFunction, feature?: OlFeature): PropertyType {
+    if (func.name === 'property') {
+      if (!feature) {
+        // TODO: throw errror instead
+        return;
+      }
+      if (isGeoStylerStringFunction(func.args[0])) {
+        return feature?.get(OlStyleUtil.evaluateStringFunction(func.args[0], feature));
+      } else {
+        return feature?.get(func.args[0]);
+      }
+    }
+
+    if (isGeoStylerStringFunction(func)) {
+      return OlStyleUtil.evaluateStringFunction(func, feature);
+    }
+    if (isGeoStylerNumberFunction(func)) {
+      return OlStyleUtil.evaluateNumberFunction(func, feature);
+    }
+    if (isGeoStylerBooleanFunction(func)) {
+      return OlStyleUtil.evaluateBooleanFunction(func, feature);
+    }
+    if (isGeoStylerUnknownFunction(func)) {
+      return OlStyleUtil.evaluateUnknownFunction(func, feature);
+    }
+    return;
+  }
+
+  public static evaluateBooleanFunction(func: GeoStylerBooleanFunction, feature?: OlFeature): boolean {
+    const args = func.args.map(arg => {
+      if (isGeoStylerFunction(arg)) {
+        return OlStyleUtil.evaluateFunction(arg, feature);
+      }
+      return arg;
+    });
+    switch (func.name) {
+      case 'between':
+        return (args[0] as number) >= (args[1] as number) && (args[0] as number) <= (args[2] as number);
+      case 'double2bool':
+        return args[0] === 0;
+      case 'in':
+        return args.slice(1).includes(args[0]);
+      case 'parseBoolean':
+        return !!args[0];
+      case 'strEndsWith':
+        return (args[0] as string).endsWith(args[1] as string);
+      case 'strEqualsIgnoreCase':
+        return (args[0] as string).toLowerCase() === (args[1] as string).toLowerCase() ;
+      case 'strMatches':
+        return new RegExp(args[1] as string).test(args[0] as string);
+      case 'strStartsWith':
+        return (args[0] as string).startsWith(args[1] as string);
+      default:
+        return true;
```

Comment:
What's the reasoning to return `true` instead of `false` by default?

Reply:
No reason. Would you prefer `false`?

Reply:
To me, it feels a little more familiar. But I do not have a strong opinion on this. So it is ok for me, if we keep it as is

Reply:
adfecb1